src/components/dialogs/shunt-compensator-creation-dialog.js

```diff
@@ -55,6 +55,7 @@ const ShuntCompensatorCreationDialog = ({
     voltageLevelOptions,
     selectedNodeUuid,
     workingNodeUuid,
+    editData,
```

Comment:
Add it in PropTypes definition.
Check if it's done for each dialog ?

Reply:
done